fastlane/Fastfile

```diff
@@ -174,6 +185,39 @@ platform :ios do
     )
   end
 
+  desc 'Download and configure code signing certificates and provisioning profiles for the Enterprise build, i.e. Prototype Builds.'
+  lane :configure_code_signing_enterprise do |options|
+    require_env_vars!(*ASC_API_KEY_ENV_VARS)
+
+    readonly = options.fetch(:readonly, true)
+
+    if readonly
+      # In readonly mode, we can use the API key.
+      #
+      # Notice the `in_house: true` parameter.
+      # Unfortunately, even using it, the API does not allow modifying certs or profiles via API key.
+      # It's here for reference and future-proofing.
+      api_key = app_store_connect_api_key(in_house: true)
```

Comment:
I discovered the `in_house` parameter in the docs while working on something else. I had my hopes up it would have spared us the trouble of authenticating locally when running for Enterprise in `readonly: false` but it didn't work...